.travis.yml

```diff
@@ -2,53 +2,52 @@ sudo: false
 language: python
 python: 2.7
 
+# This is a big repository so break Travis CI work into 2 CI jobs.
 env:
-  - CHUNK=xaa
-  - CHUNK=xab
-  - CHUNK=xac
-  - CHUNK=xad
+  - CHUNK=0
+  - CHUNK=1
 
 before_install:
   - export GALAXY_REPO=https://github.com/galaxyproject/galaxy
   - export GALAXY_RELEASE=release_16.10
   - export PLANEMO_CONDA_PREFIX="$HOME/conda"
-  - unset JAVA_HOME  # Interferes with java tools installed through conda
 
 install:
-  # html5lib (planemo dep) requires setuptools >= 18.5
-  - pip install -q flake8 planemo
+  - pip install flake8 planemo
   - planemo conda_init
-  - export PATH="$PLANEMO_CONDA_PREFIX/bin:$PATH"
-  - conda create -y -q -c bioconda --name iuc_conda samtools=0.1.19 bcftools
-  - . activate iuc_conda
+  - $PLANEMO_CONDA_PREFIX/bin/conda update -y conda
+  - $PLANEMO_CONDA_PREFIX/bin/conda config --get channels # verify conda-forge is not highest priority
+  - $PLANEMO_CONDA_PREFIX/bin/conda config --add channels conda-forge  # moves conda-forge to the top
+  - $PLANEMO_CONDA_PREFIX/bin/conda config --get channels # now conda-forge should be highest priority
```

Comment:
yep, I'll have to revert this ... it's being ignored anyway, because planemo use the condarc from ~/.planemo/condarc. Though last time I checked the order was not correct.